docs/manifest.md

```diff
@@ -1,21 +1,25 @@
-# Manifest definition file
+# Manifest store file
```

Comment:
This is NOT a manifest store, this file only defines a single manifest, a store defines a collection of manifests. I use the term manifest defintion to indicate that this is not yet a manifest, only the definition of the manifest that should be created. The manifest store will hold a set of signed manifests.

Reply:
Thanks, I reworded it accordingly.  I tried to clarify this distinction.